openclaw_hooks/nima-recall-live/ladybug_recall.py

```
@@ -0,0 +1,364 @@
+            memories.append({
+                'id': row[0],
+                'text': row[1] or "",
+                'summary': row[2] or "",
+                'who': row[3] or "unknown",
+                'layer': row[4] or "unknown",
+                'timestamp': row[5] or 0,
+                'fts_score': 1.0
+            })
+        
+        elapsed = (time.time() - start) * 1000
+        return memories, elapsed
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)